torch/csrc/jit/codegen/cuda/arith.cpp

```diff
@@ -1616,6 +1616,17 @@ TensorView* expand_as(TensorView* inp, TensorView* other) {
   return out_tensor;
 }
 
+std::vector<Val*> symbolic_sizes(TensorView* inp) {
+  auto iter_domains = inp->domain()->noReductions(inp->getMaybeRFactorDomain());
```

Comment:
Another alternative to do the same, seems clearer to me:
```suggestion
  auto iter_domains = TensorDomain::noReductions(inp->getMaybeRFactorDomain());
```